plugins/woocommerce-blocks/assets/js/blocks/product-collection/collections/related-to.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import type { InnerBlockTemplate } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { Icon, loop } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { INNER_BLOCKS_PRODUCT_TEMPLATE } from '../constants';
+import { CoreCollectionNames, LayoutOptions } from '../types';
+
+const collection = {
+	name: CoreCollectionNames.RELATED_TO,
+	title: __( 'Related Products', 'woocommerce' ),
+	icon: <Icon icon={ loop } />,
+	description: __( 'Recommend products like this one.', 'woocommerce' ),
+	keywords: [ 'related', 'product collection' ],
```

Comment:
Nitpick: words used in a title are already a keywords so we can skip `related`

```suggestion
	keywords: [ 'product collection' ],
```

Reply:
Fixed in 7aeb4052d208ffcd3ae810a466692087600e4a7f